emukit/core/loop/outer_loop.py

```diff
@@ -55,25 +55,23 @@ def __init__(self, candidate_point_calculator: CandidatePointCalculator,
         self.iteration_end_event = EventHandler()
 
     def run_loop(self, user_function: Union[UserFunction, Callable],
-                 stopping_condition: Union[StoppingCondition, int, List[StoppingCondition]],
+                 stopping_condition: Union[StoppingCondition, int],
                  context: dict = None) -> None:
         """
         :param user_function: The function that we are emulating
-        :param stopping_condition: If integer - a number of iterations to run, or an object (or list of such objects) - a stopping
-                        condition object that decides whether we should stop collecting more points. When input is a list,
-                        we stop when any of the conditions are violated.
+        :param stopping_condition: If integer - a number of iterations to run, or an object - a stopping
+                        condition object that decides whether we should stop collecting more points.
+                        Note that stopping conditions can be logically combined to represent complex stopping criteria.
```

Comment:
do you think it is useful to say here with which operators, e.g. "..logically combined (&, |) ..."

Reply:
added